standalone/standalone.py

```diff
@@ -1101,16 +1115,20 @@ def sdg_op(
 
     print("Generating synthetic dataset for:")
     print()
-    print(read_taxonomy(taxonomy, taxonomy_base))
+    print(read_taxonomy(taxonomy_path, taxonomy_base))
+
+    # Temporary measure to limit the amount of precomputed skills data used to construct the SDG dataset.
+    # Need during development to decrease training loop times and the cost of model quality.
+    set_precomputed_skills_data_ratio(sampling_size=SAMPLING_SIZE)
```

Comment:
This can be a follow-up - but we should add a bit about _why_ this isn't configurable - and how to set to the upstream default - and point to the GH issue 
something like:
```
This isn't configurable without setting SAMPLING_SIZE in this file above, because we don't want to expose this parameter. This is for testing & development only and will be removed in the near future. See [issue #154](https://github.com/redhat-et/ilab-on-ocp/issues/154)
```

Reply:
This is interesting... any idea how this change in `standalone.py` happen to be? Since I didn't change anything in the `standalone.tpl` to cause this change, did I? Can it be that the base branch has `standalone.py` out of sync?

Reply:
The change must be coming from a component then.

Reply:
Sorry. This was from #153 not running `make standealone` before submitting my PR

Reply:
Yes, this was introduced in https://github.com/redhat-et/ilab-on-ocp/pull/153. Seems like `standalone.py` was not regenerated then. Is there any change required for me to do here then?

Reply:
Tracked in https://github.com/redhat-et/ilab-on-ocp/issues/154